Audit alert v3 indicators enrichment fix (#32219)
demisto/content: Packs/PrismaCloudCompute/ReleaseNotes/1_5_1.md

#### Playbooks

##### Prisma Cloud Compute - Get Container Events

> - Fixed an error for setting events table when noe events found

Reviewer: ```suggestion
- Fixed an error for setting events table when no events are found.
```

> - Changed a task to use playbook input to retrieve WAAS events

Reviewer: ```suggestion
- Changed a task to use playbook input to retrieve WAAS events.
```

##### Prisma Cloud Compute - Audit Alert v3

> - Fixed a logic bug where all hosts results retrieved when no image or host are provided in the incident.
> ##### Prisma Cloud Compute - Audit Alert Enrichment
> 
> - Changed enrichment tasks to not extract indicators.
> ##### Prisma Cloud Compute - Audit Alert Vulnerabilities Enrichment
> 
> - Added a playbook input for controlling which indicators would get created from CVEs based on their severity. The default is "High, Critical". This would reduce the number of indicators created to avoid performance issues.

Reviewer: ```suggestion
Fixed a logic bug where all hosts results were retrieved when no image or host is provided in the incident.
##### Prisma Cloud Compute - Audit Alert Enrichment

Changed enrichment tasks to not extract indicators.
##### Prisma Cloud Compute - Audit Alert Vulnerabilities Enrichment

Added a playbook input for controlling which indicators would get created from CVEs based on their severity. The default is "High, Critical". This would reduce the number of indicators created to avoid performance issues.
```
